### PR TITLE
__DIR__ in php connector and options.managerPath for change script location

### PR DESCRIPTION
**Pull request by arswarog**
Sorry for change composer.json

### CODE REVIEW

File: connectors/php/filemanager.class.php [hunk 1]
```diff
@@ -198,14 +200,7 @@ public function getinfo() {
 		$this->item['properties'] = $this->properties;
 		$this->get_file_info('', false);
 		
-		// handle path when set dynamically with $fm->setFileRoot() method
-		if($this->dynamic_fileroot != '') {
```
Comment (simogeo) [inline connectors/php/filemanager.class.php]: Don't understand why removing this !?

File: connectors/php/filemanager.class.php [hunk 2]
```diff
@@ -35,18 +35,20 @@ class Filemanager {
...
 		// getting default config file
-		$content = file_get_contents("../../scripts/filemanager.config.js.default");
-		$config_default = json_decode($content, true);
+		$content = file_get_contents(__DIR__."/../../scripts/filemanager.config.js.default");
+		$this->config = json_decode($content, true);
 		
 		// getting user config file
-		$content = file_get_contents("../../scripts/filemanager.config.js");
-		$config = json_decode($content, true);
+        if (file_exists(__DIR__."/../../scripts/filemanager.config.js"))
```
Comment (simogeo) [inline connectors/php/filemanager.class.php]: Okay, but this well prevent merging default config to to user config files !?

**Comment by simogeo** [comment]
No worries for composer file and many thanks for pulling this here !

I'd like to be sure of what it does. I already start reviewing your code. I guess this is worth value but could you specify why and how it would be useful ?

Thanks. I'll have to test it on my part as well - not much time right now

**Comment by simogeo** [comment]
Adding a (or many) samples would be nice

File: connectors/php/filemanager.class.php (hunk 1 above)
Comment (arswarog) [inline connectors/php/filemanager.class.php]: Sorry for my English. when I tried to upload a file, you gave it to me the full path to the file system

**Comment by arswarog** [comment]
I need a separate folder for each user and what would have been the address of his roots directory
My config:        
$fm = new \Filemanager([
   "options" => [
         "serverRoot" => false,
         "fileRoot" => "/",
         "baseUrl" => false,
         "fileConnector" => Url::toRoute('api'),
         "culture" => "ru",
         "managerPath" => $fma->baseUrl.'/'
    ]
]);        
$fm->setFileRoot('statics/userfiles/'.$user->id), true);

**Comment by arswarog** [comment]
for each user their settings in accordance with his access rights the user config file I did not realized and options on the same page through window.filemanager

**Comment by cognitus** [comment]
any new with this? i need something similar

**Comment by servocoder** [comment]
I guess you could find it in [Rich Filemanager](https://github.com/servocoder/RichFilemanager).
Take a look at this [wiki article](https://github.com/servocoder/RichFilemanager/wiki/Filemanager-global-configuration). I have made that `pluginPath` option on purpose to handle 'dynamic' assets folders, which you usually have in modern frameworks. Let me know if that is what you are looking for.

**Comment by cognitus** [comment]
@servocoder  yes! thanks you

**Pull request closed without merging**